plugins/extractors/metabase/extractor.go

```diff
@@ -0,0 +1,160 @@
+package metabase
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	_ "github.com/lib/pq"
+	"github.com/odpf/meteor/core"
```

Comment:
@GrayFlash This package has been moved. Check main branch and you would need to make some changes in this.